_posts/elements/2015-04-09-buttons.md

```diff
@@ -0,0 +1,97 @@
+---
+layout:       element
+title:        Buttons
+date:         2015-04-09 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- headings
+
+slug:         buttons
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/static/sass/components/_buttons.scss"
+
+description:  Examples of the different types and variants of edX buttons.
+
+info: Buttons are great. Mkay.
+---
+
+<section class="example-set">
+    <h3>Default</h3>
```

Comment:
NIT: can we use our headings pattern classes here to render a specific heading style/level?